src/components/select/select-textbox/select-textbox.component.tsx

```diff
@@ -211,19 +201,11 @@ const SelectTextbox = React.forwardRef(
     }
 
     function getInputAriaAttributes() {
-      const joinIds = (...ids: (string | undefined)[]) =>
-        ids.filter((item) => item !== undefined).join(" ");
-      const combinedAriaLabelledBy = hasTextCursor
-        ? joinIds(
-            ariaLabelledby || labelId,
-            accessibilityLabelId || textId.current
-          )
-        : joinIds(ariaLabelledby || labelId, textId.current);
-
       return {
```

Comment:
**nitpick (non-blocking):** if this function just has a return statement after your changes, could we refactor to an arrow function to be more concise? Again not essential and I may be wrong

Reply:
it's a fair comment. I personally don't much care either way - note that because this returns an object literal it doesn't work to just do `() => { key: value, etc. }` because JS interprets the `{...}` as the function body and not for an object. It still works though if you do `() => ({...})` instead. I don't think it's better than leaving the `return` in but it's not worse either, happy to make this change if you think it's for the best.